### PR TITLE
Processed amount

### PR DESCRIPTION
**Pull request by kpbacode**
This PR resolve fallowing issue:
https://github.com/killbill/killbill-admin-ui/issues/322

### CODE REVIEW

File: app/models/kaui/transaction.rb [hunk 1]
```diff
@@ -34,7 +34,17 @@ def amount_to_money
...
+  def processed_amount_to_money
+    Kaui::Base.to_money(processed_amount, currency)
+  end
+
   def self.amount_to_money(transaction)
     self.new(:amount => transaction.amount, :currency => transaction.currency).amount_to_money
   end
+
+  def self.processed_amount_to_money(transaction)
+    self.new(:processed_amount => transaction.processed_amount, :currency => transaction.currency).processed_amount_to_money
```
Comment (pierre) [inline app/models/kaui/transaction.rb:47]: Needs to be `:processed_currency => transaction.processed_currency` (and `processed_amount_to_money` above updated as well).

File: app/views/kaui/payments/_payment_table.html.erb [hunk 2]
```diff
@@ -96,8 +96,20 @@
...
             <td>
               <% if transaction.amount.present? %>
+                  <% if transaction.amount != transaction.processed_amount %>
+                  <%= humanized_money_with_symbol Kaui::Transaction.processed_amount_to_money(transaction) %>
+                  (<%= transaction.currency %>)
+                  <a href="#" data-toggle="tooltip" class="kb-tooltip" title="<%= humanized_money_with_symbol Kaui::Transaction.amount_to_money(transaction) %>(<%= transaction.currency %>)">(?)</a>
+                  <div class="tooltip bs-tooltip-top" role="tooltip">
+                    <div class="arrow"></div>
+                    <div class="tooltip-inner">
+                      <%= humanized_money_with_symbol Kaui::Transaction.amount_to_money(transaction) %>(<%= transaction.currency %>)
```
Comment (pierre) [inline app/views/kaui/payments/_payment_table.html.erb:106]: Maybe we should show a legend, to make it clear, e.g. `Requested amount: <%= humanized_money_with_symbol Kaui::Transaction.amount_to_money(transaction) %> (<%= transaction.currency %>)`?

**Review by pierre — CHANGES_REQUESTED** [review]

File: app/models/kaui/transaction.rb (hunk 1 above)
Comment (kpbacode) [reply app/models/kaui/transaction.rb]: Screenshot 2022-10-14 at 11 04 07

Right...

File: app/views/kaui/payments/_payment_table.html.erb (hunk 2 above)
Comment (kpbacode) [reply app/views/kaui/payments/_payment_table.html.erb]: Screenshot 2022-10-14 at 11 45 52

**Review by pierre — APPROVED** [review]

**Pull request merged**